src/components/tools/tool-tile.sass

```diff
@@ -42,6 +42,30 @@
     &:focus
       outline: none
 
+  .tool-tile-resize-handle
+    position: absolute
+    right: -2px
+    bottom: -2px
+    box-sizing: border-box
+    width: 24px
+    height: 24px
+    z-index: 1
```

Comment:
I think the geometry tile uses z-indices up to 9 or so, so you may want to make this 10 to make sure it shows up on top of points and other geometry elements.